presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -79,6 +80,7 @@
     private HiveCompressionCodec hiveCompressionCodec = HiveCompressionCodec.GZIP;
     private boolean respectTableFormat = true;
     private boolean immutablePartitions;
+    private String insertExistingPartitionsBehavior;
```

Comment:
Please use enum.

See how `hiveCompressionCodec` is implemented above.

By default it should be set to `APPEND`

Reply:
done.